packages/helix_editor.rb

```diff
@@ -0,0 +1,73 @@
+require 'package'
+
+class Helix_editor < Package                 # The first character of the class name must be upper case
+  description 'A vim like editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/refs/tags/22.12.tar.gz'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972'   # Use the command "sha256sum"
+  #source_url 'file:///usr/local/home/helix/deleteme.tar.gz'
+  #source_sha256 'c3a0e022a7bcd6b139ae7abf1e0dae33c8f9ec82dc680076d9e5ea6593b85bc6'
+
+  depends_on 'rust' => :build
+
+  # TODO is it the appropriate location?
+  # By default, helix expects runtime to be in ~/.config or XDG_CONFIG_HOME.
+  # The first location is not appropriate because helix will not be installed in ~/
+  # With the second location crew complaints that it is not  FHS3 compliant.
+  @runtime_dest_dir_suffix = "/var/lib/helix"
+
+  def self.build
+   puts "Building. This may be long."
+   system "cargo build \
+      --release \
+      --locked \
+     "
+  end
+
+  def self.install
+    # Copy executable
+    helix_executable_dest_dir = "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.mkdir_p("#{helix_executable_dest_dir}")
+    FileUtils.cp("./target/release/hx", "#{helix_executable_dest_dir}")
+    # Copy runtime dir
+    FileUtils.mkdir_p("#{CREW_DEST_PREFIX}#{@runtime_dest_dir_suffix}")
+    FileUtils.cp_r("./runtime", "#{CREW_DEST_PREFIX}#{@runtime_dest_dir_suffix}")
+    # Set env var for runtime dir
+    env_dir = "#{CREW_DEST_PREFIX}/etc/env.d"
+    FileUtils.mkdir_p "#{env_dir}"
+    File.write("#{env_dir}/helix",
+               "export HELIX_RUNTIME=#{CREW_PREFIX}#{@runtime_dest_dir_suffix}/runtime")
```

Comment:
If this is a single executable without any dependencies, it could be installed directly in `"#{CREW_DEST_PREFIX}/bin"`.  The installed location which is `/usr/local/bin` by default is already in the path.

Reply:
I removed this code now. I do not need to set the environment variable anymore.